framework/docs/source/ref-simulation-deployment-runtimes.rst

```diff
@@ -5,17 +5,15 @@
 Simulation vs. Deployment Runtime
 =================================
 
-From both a developer and user experience perspective, the only change required when
-moving from a simulated to a real-world Flower federation is to set the correct
-federation address pointing to a SuperLink deployed either in a `simulation
-<how-to-run-simulations.html>`_ or `deployment
-<how-to-run-flower-with-deployment-engine.html>`_ runtime. In this way, the same
-application developed using the Flower simulation runtime can be directly deployed to a
-`real-world Flower federation <explanation-flower-architecture.html>`_. Additionally,
-the `Flower CLI <ref-api-cli.html>`_ remains the same across both environments, ensuring
-a smooth transition without the need for additional configuration or tooling changes.
-The following table outlines the key characteristics that differentiate simulated Flower
-federations from deployed ones.
+Flower Apps can be executed with both with `simulation <how-to-run-simulations.html>`_
+or `deployment <how-to-run-flower-with-deployment-engine.html>`_ runtimes. Switching
+between runtimes simply requires specifying a different type of `federation` when
+executing the `flwr run` command via the `Flower CLI <ref-api-cli.html>`_.
+
+While Flower Apps can run in both simulation and deployment without making changes to
+their code, there are some differences on how they get executed depending on the
+runtime. The following table outlines the key characteristics that differentiate
+simulated Flower federations from deployed ones.
```

Comment:
```suggestion
runtime. The following table outlines the key characteristics that differ when
executing a FlowerApp in a Flower federation with the simulation runtime from another using the deployment runtime.
```

Reply:
@dstripelis I think like this is clearer. Technically speaking federations aren't executed.